backend/.env.example

```
@@ -6,3 +6,11 @@ REDIS_URL="redis://localhost:6379"
 PINECONE_API_KEY=
 DATABASE_URL=
 DATABASE_URL_UNPOOLED=
+
+# Clerk Configuration
+CLERK_SECRET_KEY=sk_test_YCNvZKpmGrqcwziGhGuH7Pv99X2DDi8YxFthC6qDvX
+CLERK_WEBHOOK_SIGNING_SECRET=whsec_123
+CLERK_WEBHOOK_SECRET=whsec_LxltdYfWqllWpT8Tab9q4KxmNPKdCmei
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CLERK_WEBHOOK_SECRET key should go before the CLERK_WEBHOOK_SIGNING_SECRET key

(UnorderedKey)